test/test_remap.py

```diff
@@ -257,8 +331,93 @@ def test_overwrite(self):
         destination_dataset = ux.open_dataset(gridfile_geoflow, dsfile_v1_geoflow)
 
         # Perform Remapping
-        remap_to_edge_centers = source_grid['v1'].remap.inverse_distance_weighted(destination_grid=destination_dataset.uxgrid,
-                                                                         remap_to="nodes")
+        remap_to_edge_centers = source_grid['v1'].remap.inverse_distance_weighted(
+            destination_grid=destination_dataset.uxgrid,
+            remap_to="face centers", coord_type='cartesian')
 
         # Assert the remapped data is different from the original data
         assert not np.array_equal(destination_dataset['v1'], remap_to_edge_centers)
+
+    def test_source_data_remap(self):
+        """Test the remapping of all source data positions."""
+
+        # Open source and destination datasets to remap to
+        source_uxds = ux.open_dataset(mpasfile_QU, mpasfile_QU)
+        destination_dataset = ux.open_dataset(gridfile_geoflow, dsfile_v1_geoflow)
+
+        # Remap from `face_centers`
+        face_centers = source_uxds['latCell'].remap.inverse_distance_weighted(
+            destination_grid=destination_dataset.uxgrid,
+            remap_to="nodes"
+        )
+
+        # Remap from `nodes`
+        nodes = source_uxds['latVertex'].remap.inverse_distance_weighted(
+            destination_grid=destination_dataset.uxgrid,
+            remap_to="nodes"
+        )
+
+        # Remap from `edges`
+        edges = source_uxds['angleEdge'].remap.inverse_distance_weighted(
+            destination_grid=destination_dataset.uxgrid,
+            remap_to="nodes"
```

Comment:
Switch from ux.open_dataset to ux.open_grid where possible, replacing destination_dataset.uxgrid with destination_grid afterwards.